cloudevents/http/conversion.py

```diff
@@ -0,0 +1,52 @@
+import typing
+
+from cloudevents.conversion import from_dict as _abstract_from_dict
+from cloudevents.conversion import from_http as _abstract_from_http
+from cloudevents.conversion import from_json as _abstract_from_json
+from cloudevents.conversion import to_binary, to_dict, to_json, to_structured  # noqa
+from cloudevents.http.event import CloudEvent
+from cloudevents.sdk import types
+
+
+def from_json(
+    data: typing.Union[str, bytes],
+    data_unmarshaller: types.UnmarshallerType = None,
+) -> CloudEvent:
+    """
+    Cast json encoded data into an CloudEvent
+    :param data: json encoded cloudevent data
+    :param data_unmarshaller: Callable function which will cast data to a
+        python object
+    :type data_unmarshaller: typing.Callable
+    :returns: CloudEvent representing given cloudevent json object
+    """
```

Comment:
Please use docs from the `cloudevents.conversion`. I've already fixed and reviewed all of them in the previous PR.

Applies to the other methods in this file as well.